app/models/ems_folder.rb

```diff
@@ -177,7 +162,7 @@ def folder_path_objs(*args)
     options = args.extract_options!
     folders = path(:of_type => "EmsFolder")
     folders = folders[1..-1] if options[:exclude_root_folder]
-    folders = folders.reject { |f| NON_DISPLAY_FOLDERS.include?(f.name) } if options[:exclude_non_display_folders]
+    folders = folders.reject(&:hidden?) if options[:exclude_non_display_folders]
```

Comment:
Ok, so we added the hidden column for ems_folders and we set it on refresh but what about here for existing ems_folders?  Can hidden be NULL in the database at this point or wherever we call EmsFolder#hidden?

We do this several places.... does it make sense to migrate the database to convert all existing folders to either hidden true or false and let refresh update it later on?